scripts/generate_expected_bpf2c_output.ps1

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) Microsoft Corporation
+# SPDX-License-Identifier: MIT
+
+# Whenever either the BPF2C code changes or any of the sample programs changes,
+# bpf2c_tests will fail unless the expected files in tests\bpf2c_tests\expected
+# are updated. This script can be used to regenerate the expected files.
+#
+# Usage:
+# .\scripts\generate_expected_bpf2c_output.ps1 <build_output_path>
+# Example:
+# .\scripts\generate_expected_bpf2c_output.ps1 .\x64\Debug\
+
+function filter_and_emit_output
+{
+    param([String[]] $input_buffer, [String]$output_file)
+
+    $output_buffer = New-Object System.Collections.ArrayList
+    foreach ($line in $input_buffer)
+    {
+        if (!$line.StartsWith("#line"))
+        {
+            [void]$output_buffer.Add($line)
+            continue
+        }
+
+        if ($line.IndexOf('"') -eq -1)
+        {
+            [void]$output_buffer.Add($line)
+            continue
+        }
+
+        [void]$output_buffer.Add($line.Substring(0, $line.IndexOf('"') + 1) +   $line.Substring($line.LastIndexOf('\\') + 2));
+    }
+
+    $output_buffer > $output_file
+
+    # Change output file format to UTF-8
+    (Get-Content -path $output_file) | Set-Content -Encoding Ascii -Path $output_file
+}
+
+function generate_expected_output
+{
+    param([string]$build_path)
+
+    $ps_root = $PSScriptRoot
+    $sample_path = $ps_root + "\..\tests\sample"
+    $expected_output_path = $ps_root + "\..\tests\bpf2c_tests\expected"
+    $bpf2c_command = ".\bpf2c.exe"
+
+    $current_location = Get-Location
+
+    ## Get all files in the sample path
```

Comment:
please add generated files for sock_addr and sockops.c

Reply:
Added.